modules/ROOT/pages/domain-based-application-cookbook.adoc

```diff
@@ -0,0 +1,154 @@
+= Test Domain-Based Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MUnit 2.x allows you to write MUnit test suites for your Mule applications that share their resources with a Mule domain without the need for any extra configurations.
+
+== Base Mule Application
+
+Consider a Mule domain called `domain-config` that defines a global configuration for the HTTP listeners of all the Mule applications under it:
```

Comment:
```suggestion
Consider a Mule domain called `domain-config` that defines a global configuration for the HTTP listeners of all the Mule applications within it:
```
I think things are "in" a domain rather than "under" it?